docs/sequences/e2ee-sequences.md

```
@@ -0,0 +1,32 @@
+2. New device -> QR payload: base64url(JSON { device_pubkeys, nonce: N, ts, sig: sign_D(N||ts) })
+3. Trusted device scans QR
+4. Both devices derive SAS from transcript (hash of {identity_pubkey, device_pubkeys, N, ts}) -> 7 emojis
+5. Human verifies SAS match
+6. Trusted device signs attestation: sign_T({ user_identity_pub, device_pubkeys, N, ts })
+7. Trusted device -> Server: POST /devices/attest(attestation)
+8. Server records device; broadcasts WS event
+
+## 2) File Share (Encrypt + Upload + Share DEK)
+1. Client picks file; gen random DEK
+2. For each chunk i:
+   - nonce_i = HKDF(DEK, info="file-chunk"||i)[0..12]
```

markdownlint-cli2 (0.17.2)

Reversed link syntax
(DEK, info="file-chunk"||i)[0..12]

(MD011, no-reversed-links)